triaging.rst

```diff
@@ -47,6 +47,73 @@ the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
    Existing members with the "Developer" role on the `issue tracker`_ are welcome to
    self-nominate for triage team membership if they are interested.
+   
+
+GitHub Labels 
+'''''''''''''
+
+An important component of triaging PRs and issues on GitHub involves
+appropriately categorizing them through the usage of labels.
```

Comment:
Projects other than CPython, such as devguide, may have issues enabled for GitHub. These projects may address labels for issues in the CONTRIBUTING file or README file.

Reply:
Was this suggestion intended to be an explanation for "and issues on GitHub" => "for the CPython repo" or is it a separate suggestion to mention where someone could find a guide for the usage of labels in other Python projects?

Reply:
This was intended as a comment for the CPython repo.